src/klipse/utils.cljs

```diff
@@ -9,6 +9,29 @@
    [cljs.core.async :refer [timeout <! chan put!]]
    [cemerick.url :refer [url]]))
 
+(defn fetch-shortened-url
+  "Calls is.gd with the current URL and returns
+  a shortened version for the user to copy."
+  [current]
+  (go (let [response (<! (http/get
+      "https://cors-anywhere.herokuapp.com/https://is.gd/create.php"
+      {:with-credentials? false
+      :headers {"accept" "application/json" "content-type" "application/json"}
+      :query-params {:format "simple" :url "https://google.com"}}))]
+      (if-not (= (:status response) 200)
+        (
```

Comment:
better to move alert and print into a separate function

Reply:
Done, although I can't seem to get the URL to output on to the Klipse output. Is there another function I need to call besides `print` to make that happen?